src/components/calcite-time-picker/calcite-time-picker.scss

```diff
@@ -108,3 +108,7 @@
     }
   }
 }
+
+:host([lang="ar"], [lang="he"]) {
```

Comment:
@anveshmekala this will only work if the component has a lang attribute on it. What if the lang is set on a parent?

Also, we don't want to set `direction` on the host because it can be overriden by a user.

So, if we need to force direction ltr always, lets set it inside the component where the direction needs to be ltr always.

Reply:
If the user doesn't specify the `lang` attribute then the requirement to match the `rtl` with `ltr` will be out of scope.

Reply:
I guess my question is why isn't the browser correctly setting the direction for `ar` and `he` locales?  Are we trying to override otherwise correct behavior on the part of the browser or is this a browser bug?

Reply:
And I agree with Matt's comments.  Any kind of CSS solution is brittle and overridable.  If we need to address direction it needs to be in the component itself.  I also would like to see this component go away in the future in favor of the new `calcite-input-time` component which would likely just consist of an input.

Reply:
> Any kind of CSS solution is brittle and overridable.

Only if its on the host. ;)

> Are we trying to override otherwise correct behavior on the part of the browser

Thats what the issue wants done. I'm not sure why exactly.

Reply:
1. We would not want to check for specific languages and set styles on any components.
2. Setting styles on the host will expose those styles to the user and allow them to easily override them.
3. Using this code above, the following wouldn't set the `direction: ltr;` css.

```html
<div lang="ar">
<calcite-time-picker />
</div>
```

So to handle this issue, we can:

Set `direction:ltr` on the element that contains the row of time picking inputs.

This would ensure that it is always LTR even if the direction is set to RTL.